### DISCUSSION TITLE
What are the GPU requirements for Bertopic?

### DISCUSSION THREAD

**Discussion by akarshghale** [MaartenGr/BERTopic#1196] [Q&A]
Hi folks,

I am planning to host a server for automating Bertopic tasks and would like to know what are the minimum GPU requirements and do you know any cheap GPU server providers?

Thank you!

**Comment by MaartenGr** [comment]
That highly depends on your configuration of BERTopic. For example, the underlying embedding model typically requires a GPU but if you were to use TF-IDF instead, you would not need a GPU.
